### PR TITLE
chore: add unit test to weather api

### PR DESCRIPTION
**Pull request by fernandoox**
## Purpose
It's necessary to improve our coverage :chart_with_upwards_trend: 

## Approach
- I moved the file weatherApi.ts
- I wrote the unit test for the file 👆️
image
image
## Where should the reviewer start?
Unit Test

## This PR addresses a:
- [ ] Bug
- [ ] Feature
- [X] Technical Debt
- [ ] Chore
- [ ] Vulnerability

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -16,9 +16,9 @@
     "module": "esnext",
     "moduleResolution": "node",
     "resolveJsonModule": true,
-    "isolatedModules": true,
+    "isolatedModules": false,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
```
Comment (ticsaul) [inline tsconfig.json]: I think you should remove this comma

File: .idea/inspectionProfiles/Project_Default.xml [hunk 2]
```diff
@@ -2,5 +2,10 @@
   <profile version="1.0">
     <option name="myName" value="Project Default" />
     <inspection_tool class="Eslint" enabled="true" level="WARNING" enabled_by_default="true" />
+    <inspection_tool class="SpellCheckingInspection" enabled="false" level="TYPO" enabled_by_default="false">
```
Comment (ticsaul) [inline .idea/inspectionProfiles/Project_Default.xml]: Is neccesary this file ?

**Review by ticsaul — CHANGES_REQUESTED** [review]
I write comments in some files.

File: tsconfig.json (hunk 1 above)
Comment (fernandoox) [reply tsconfig.json]: I ran the prettier, done

**Review by ticsaul — APPROVED** [review]
Ok

**Review by oscarmtl7 — COMMENTED** [review]
It´s okay, only can you check the yarn file, i think have it conflict

**Pull request merged**